src/tools/clippy/clippy_config/src/msrvs.rs

```diff
@@ -17,7 +17,8 @@ macro_rules! msrv_aliases {
 
 // names may refer to stabilized feature flags or library items
 msrv_aliases! {
-    1,81,0  { LINT_REASONS_STABILIZATION }
+    1,82,0 { CONST_EXTERN_FN }
+    1,81,0 { LINT_REASONS_STABILIZATION }
```

Comment:
if anyone from clippy is following along, is there some equivalent of `CURRENT_RUSTC_VERSION` here? In any case given that we're close to a rust release this may or may not be accurate when merged